src/main/components/PianoRoll/PianoRoll.tsx

```diff
@@ -127,8 +128,14 @@ const PianoRollWrapper: FC = observer(() => {
     (e: React.WheelEvent) => {
       if (e.altKey || e.ctrlKey) {
         // zooming
-        const scaleFactor = isTouchPadEvent(e.nativeEvent) ? 0.01 : 0.05
-        s.scaleX = Math.max(SCALE_X_MIN, s.scaleX + e.deltaY * scaleFactor)
+        let scaleFactor = undefined
+        if (navigator.appVersion.indexOf("Mac") != -1) { // MacOS
```

Comment:
This is not really needed, but pinch-to-zoom on a Mac is not classified as an `isTouchPadEvent` and making it negative makes it zoom in the natrual direction